test/scripts/install_test_deps.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+install_gcloud()
```

Comment:
Also we would want to split out the functions from them being called, i,e. the codeblock I mentioned above would be in `test/` because it wouldn't be called by an external repo, only in our tests (since those environment variables are specific to our implementation)